psng/python/tool_measurement.py

```diff
@@ -182,17 +190,14 @@ def on_btn_probe_workpiece_released(self, gtkbutton, data=None):
         self.spbtn_block_height.set_value(float(a[2]))
         self.add_history(gtkbutton.get_tooltip_text(), "Z", z=a[2])
 
-    # Down probe to table for measuring it and use for calculate tool setter height and can set G10 L20 Z0 if you tick auto zero
-    def on_btn_probe_table_released(self, gtkbutton, data=None):
-        # Start psng_probe_table.ngc
-        if self.ocode("o<psng_probe_table> call") == -1:
-            return
-        a = self.probed_position_with_offsets()
-        self.add_history(gtkbutton.get_tooltip_text(), "Z", z=a[2])
-        self.set_zerro("Z", 0, 0, a[2])
-
     # Probe tool Diameter
-    def on_tool_dia_released(self, gtkbutton, data=None):
+    def on_btn_tool_dia_released(self, gtkbutton, data=None):
+        tooltable = self.inifile.find("EMCIO", "TOOL_TABLE")
+        if not tooltable:
+            self.error_dialog(
+                "Tool Diameter Measurement Error",
+                secondary="Did not find a toolfile file in [EMCIO] TOOL_TABLE",
+            )
```

Comment:
Why was this added? Does the method not work if a tool table file is not specified? As I understand it, this will work just fine without the tool table (I've not tested this understanding!)